src/main/kotlin/org/rust/ide/structure/RustConstTreeElement.kt

```diff
@@ -6,7 +6,7 @@ import org.rust.lang.core.psi.RustConstItemElement
 
 class RustConstTreeElement(element: RustConstItemElement) : PsiTreeElementBase<RustConstItemElement>(element) {
 
-    override fun getPresentableText(): String? = element?.name
+    override fun getPresentableText() = "${element?.name}: ${element?.type?.text}"
```

Comment:
OK, I can fix that. I don't know what `pin` is, but I assume the same fix must be applied to all places where the type information is displayed in the Project Structure (consts, statics, function arguments, enums with tuple-like data attached). Is it correct?